src/modules/elements/google-map/element.js

```diff
@@ -367,6 +368,16 @@ export default class GoogleMap extends Component {
 		);
 	}
 
+	renderIframe() {
+		return (
+			<Iframe
+				url="https://www.google.com/maps/embed?pb=!1m10!1m8!1m3!1d65369183.36050215!2d0!3d0!3m2!1i1024!2i768!4f13.1!5e0!3m2!1sen!2sus!4v1633005420084!5m2!1sen!2sus" // eslint-disable-line max-len
+				width="450px"
```

Comment:
Do we have a style to cover smaller mobile widths?
I know this is the admin, but still... a `max-width: 100%` could go a long way toward avoiding frustration and a support request for a too-wide iframe in a narrow page? (sounds like a side-scroll nightmare?)

Reply:
@Camwyn We currently don't have a mobile design for this. I believe this will be addressed in the next version of the re-skin. Ideally we should have the iframe in a single column same as on the frontend.